internal/cli/iam/projects/invitations/list_test.go

```diff
@@ -37,7 +37,7 @@ func TestList_Run(t *testing.T) {
 
 	mockStore.
 		EXPECT().
-		ProjectInvitations(listOpts.ConfigOrgID(), listOpts.newInvitationOptions()).
+		ProjectInvitations(listOpts.ConfigProjectID(), listOpts.newInvitationOptions()).
```

Comment:
👀  can you please start adding e2e testing, I think all of org invites is testable now, and listing should testable as well, even if it's to return an empty list

Somebody added this file and I think we have forgotten about it https://github.com/mongodb/mongocli/blob/master/e2e/E2E-TESTS.md

Reply:
I have created this task for the e2e tests https://jira.mongodb.org/browse/CLOUDP-93856. I was planning to add them once I have added also the project commands.

Reply:
we should not wait long between adding commands and adding e2e tests, I sometimes forget to check and let them slide as it's hard to test create without a delete in place